httpbin/handlers.go

```diff
@@ -178,31 +178,31 @@ var (
   ]
 }
 `)
-	statusHTTP300body = []byte(`<!doctype html>
+	statusHTTP300body := []byte(fmt.Sprintf(`<!doctype html>
 <head>
 <title>Multiple Choices</title>
 </head>
 <body>
 <ul>
-<li><a href="/image/jpeg">/image/jpeg</a></li>
-<li><a href="/image/png">/image/png</a></li>
-<li><a href="/image/svg">/image/svg</a></li>
+<li><a href="%[1]s/image/jpeg">/image/jpeg</a></li>
+<li><a href="%[1]s/image/png">/image/png</a></li>
+<li><a href="%[1]s/image/svg">/image/svg</a></li>
```

Comment:
Wow, TIL about this `%[1]s` syntax, that's very handy!

Reply:
:smile: Would be possible to use this also use this instead of the templates if that dependency should be avoided. But I do not think this would make the code cleaner and all `%` would have to be escaped.